Esri/arcgis-maps-sdk-swift-samples
# [New] Match viewpoint of geo views

## Description

This PR implements `Match viewpoint of geo views` in `Scenes` category.
URL to README: [README](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-MatchViewpointOfGeoviews/Shared/Samples/Match%20viewpoint%20of%20geo%20views)

## Linked Issue(s)

- `swift/issues/5282`

## How To Test

- Pan and zoom on the map view or scene view to see the other view's viewpoint match the one interacted with.

## Screenshots

![match-viewpoint-of-geo-views](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/e288825d-da58-42f1-a526-30700d4f75a9)
